editor/editor_node.cpp

```diff
@@ -2450,15 +2450,19 @@ void EditorNode::_run(bool p_current, const String &p_custom) {
 
 	emit_signal(SNAME("play_pressed"));
 	if (p_current) {
+		run_current_filename = run_filename;
 		play_scene_button->set_pressed(true);
 		play_scene_button->set_icon(gui_base->get_theme_icon(SNAME("Reload"), SNAME("EditorIcons")));
+		play_scene_button->set_tooltip(TTR("Reload the played scene."));
 	} else if (!p_custom.is_empty()) {
 		run_custom_filename = p_custom;
 		play_custom_scene_button->set_pressed(true);
 		play_custom_scene_button->set_icon(gui_base->get_theme_icon(SNAME("Reload"), SNAME("EditorIcons")));
+		play_custom_scene_button->set_tooltip(TTR("Reload the played scene."));
 	} else {
 		play_button->set_pressed(true);
 		play_button->set_icon(gui_base->get_theme_icon(SNAME("Reload"), SNAME("EditorIcons")));
+		play_button->set_tooltip(TTR("Reload the played scene."));
```

Comment:
I guess this one is more "Reload the main scene."? Though the current description is not _wrong_ either.

Reply:
I just used the same string everywhere.